platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -1039,6 +1048,9 @@
 				35599DB81D46AD7F0048254D /* Categories */,
 				353933F01D3FB6BA003F57D7 /* Layers */,
 				35136D491D4277EA00C20EFD /* Sources */,
+				1F0666881EC64F8E001C16D7 /* MGLLight.h */,
+				1F0666891EC64F8E001C16D7 /* MGLLight.mm */,
+				1F7454941ECD450D00021D39 /* MGLLight_Private.h */,
```

Comment:
Please make corresponding changes to the macOS project and umbrella header to unbreak the build.